internal/services/compute/virtual_machine_run_command_resource.go

```diff
@@ -0,0 +1,702 @@
+						ValidateFunc: validation.IsUUID,
+						ConflictsWith: []string{
+							"error_blob_managed_identity.0.object_id",
+						},
+					},
+					"object_id": {
+						Type:         pluginsdk.TypeString,
+						Optional:     true,
+						Sensitive:    true,
+						ValidateFunc: validation.IsUUID,
+						ConflictsWith: []string{
+							"error_blob_managed_identity.0.client_id",
+						},
+					},
+				},
+			},
+		},
+
+		"error_blob_uri": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			ValidateFunc: validation.IsURLWithHTTPS,
+		},
+
+		"output_blob_managed_identity": {
+			Type:      pluginsdk.TypeList,
+			Optional:  true,
+			MaxItems:  1,
+			Sensitive: true,
+			RequiredWith: []string{
+				"output_blob_uri",
+			},
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"client_id": {
+						Type:         pluginsdk.TypeString,
+						Optional:     true,
+						Sensitive:    true,
+						ValidateFunc: validation.StringIsNotEmpty,
+						ConflictsWith: []string{
+							"output_blob_managed_identity.0.object_id",
+						},
+					},
+					"object_id": {
+						Type:         pluginsdk.TypeString,
+						Optional:     true,
+						Sensitive:    true,
+						ValidateFunc: validation.StringIsNotEmpty,
+						ConflictsWith: []string{
+							"output_blob_managed_identity.0.client_id",
+						},
+					},
+				},
+			},
+		},
+
+		"output_blob_uri": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			ValidateFunc: validation.IsURLWithHTTPS,
+		},
+
+		"parameter": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"name": {
+						Type:         pluginsdk.TypeString,
+						Required:     true,
+						ValidateFunc: validation.StringIsNotEmpty,
+					},
+					"value": {
+						Type:         pluginsdk.TypeString,
+						Required:     true,
+						ValidateFunc: validation.StringIsNotEmpty,
+					},
+				},
+			},
+		},
+
+		"protected_parameter": {
+			Type:      pluginsdk.TypeList,
+			Optional:  true,
+			Sensitive: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"name": {
+						Type:         pluginsdk.TypeString,
+						Required:     true,
+						Sensitive:    true,
+						ValidateFunc: validation.StringIsNotEmpty,
+					},
+					"value": {
+						Type:         pluginsdk.TypeString,
+						Required:     true,
+						Sensitive:    true,
+						ValidateFunc: validation.StringIsNotEmpty,
+					},
+				},
+			},
+		},
+
+		"run_as_password": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			Sensitive:    true,
+			ValidateFunc: validation.StringIsNotEmpty,
+		},
+
+		"run_as_user": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			ValidateFunc: validation.StringIsNotEmpty,
+		},
+
+		"timeout_in_seconds": {
+			Type:         pluginsdk.TypeInt,
+			Optional:     true,
+			ValidateFunc: validation.IntAtLeast(1),
+		},
```

Comment:
Terraform has timeouts which are used for this purpose, whilst I understand this value is being passed to the API - it's another example of why this resource probably isn't a fit for Terraform.

Reply:
this property is hide in schema now, in API this field is set to the resource timeout for create/update